src/screens/LinkProject/LinkProject.tsx

```diff
@@ -123,7 +123,13 @@ function SelectProject({
   const [isSelectingProject, setSelectingProject] = useState(false);
 
   const handleSelectProject = React.useCallback(
-    (project: SelectProjectsQueryQuery["viewer"]["accounts"][number]["projects"][number]) => {
+    (
+      project: NonNullable<
+        NonNullable<
+          NonNullable<SelectProjectsQueryQuery["viewer"]>["accounts"][number]["projects"]
+        >[number]
+      >
```

Comment:
@ghengeveld this one is particulary bad. I'm not sure why it's possible to have null results in the `accounts.projects` array.

Reply:
This gets into permissions territory. `null` is also returned if you don't have access to a resource, so as not to reveal its existence. However, it doesn't make sense to include null values in an array, so that seems wrong.